app/controllers/repo/config.js

```diff
@@ -0,0 +1,6 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+  queryParams: ['status'],
+  status: 'customize'
```

Comment:
If it's the same `status` as in component, then it'd be better to:

```js
import { STATUSES } from 'travis/components/request/configs'

export default Controller.extend({
  status: STATUSES.CUSTOMIZE
})
```
In this case it'll always keep in sync and it won't be needed to find/replace hardcoded strings.

Reply:
nice! i didn't know i could do that ... seems obvious now :)

curiously, on the final branch this status is entirely gone. i believe this may have been a left over from previous refactoring, and i may have deleted it in the meantime.